.github/workflows/build.yml

```diff
@@ -89,7 +89,7 @@ jobs:
             ) &&
             !cancelled()
           }}
-        run: ${CG_EXEC} snyk test --all-projects --severity-threshold=high --policy-path=.snyk --json-file-output=snyk-test.json --org=hiero-client-sdks
+        run: ${CG_EXEC} snyk test --all-projects '--configuration-matching=^(?!_internal-).*' --severity-threshold=high --policy-path=.snyk --json-file-output=snyk-test.json --org=hiero-client-sdks
```

Comment:
Exclude "_internal-*" that the Android plugin automatically adds to the Android example. Snyk currently fails although we do not use these: There are no tests in the example (but Snyk still does "catch all" by default).